tracing-attributes/src/lib.rs

```diff
@@ -353,13 +353,31 @@ mod expand;
 /// const fn my_const_function() {}
 /// ```
 ///
+/// When a function returns a [`pin`]ned future and the function and its future have to be inside the
+/// same span, current span can be attached to the returned future like this:
+///
+/// ```
+/// # use tracing_attributes::instrument;
```

Comment:
This doesn't compile, you need to import the `tracing::Instrument` trait. When you're at it, maybe also sort the imports in the order std -> tracing ->  tracing_subscriber. Thanks!